tests/test_insert_assert.py

```diff
@@ -2,13 +2,13 @@
 import sys
 
 import pytest
+from pytest_pretty import ansi_escape
 
 from devtools.pytest_plugin import load_black
 
 pytestmark = pytest.mark.skipif(sys.version_info < (3, 8), reason='requires Python 3.8+')
 
 
-config = "pytest_plugins = ['devtools.pytest_plugin']"
```

Comment:
I was getting `ValueError: Plugin already registered: devtools.pytest_plugin` and removing this fixed it for me locally.